### ISSUE TITLE
mouse shape changes not announced in windows 10

### ISSUE THREAD

**Issue by JamaicanUser** [nvaccess/nvda#5595]
Using Windows 10, 64 bit, The report mouse shape changes and enable mouse tracking options checked. NVDA is not announcing the shape change as the mouse pointer is moved. I don't know if this applies to any other variant of Windows, though.

**Comment by jcsteh** [comment]
Confirmed.

It looks like this broke in two phases:

1. #3801 (NVDA 2014.2): mouseHandler only reports the shape after it is pumped 10 times to avoid flooding. However, we no longer pump constantly, only on demand. This needs to be changed to a timer instead.
2. #5302 (NVDA 2015.4): We seem to be missing most cursor shape changes. I'm pretty sure this is because these get blocked by eventHandler.shouldAcceptEvent. The fix is probably to allow events from the desktop window.

**Comment by nvaccessAuto** [comment]
Incubated in ae3b58618b0c51fbb9d4321e92bd1ec301231e67.
